### ISSUE TITLE
Should onPointerDown event on DropdownMenuTrigger replaced with onClick handler?

### ISSUE THREAD

**Issue by mellosti** [radix-ui/primitives#3012]
I am trying to use the `DropdownMenu` on a component that can be dragged. However, the `onPointerDown` handler (see [here](https://github.com/radix-ui/primitives/blob/main/packages/react/dropdown-menu/src/DropdownMenu.tsx#L116)) on the `DropdownMenuTrigger` prevents the trigger from being dragged. I replaced it with onClick and solved the issue locally. I can now both drag the trigger and also click to open the dropdown.

Why is `onPointerDown` used instead of `onClick`? Can we replace it?

My code:

```js
    <DropdownMenu>
      <DropdownMenuTrigger asChild>
       <button draggable={true} onDrag={onDrag} ...>Drag me!</button>
      </DropdownMenuTrigger>
      <DropdownMenuContent>
          ...
      </DropdownMenuContent>
    </DropdownMenu>
```

**Comment by duhaime** [comment]
@chaance @hadihallak is this library still maintained?

**Comment by chaance** [comment]
Yes, it's still maintained. This behavior is by design. Opening the menu on pointerdown more closely mirrors behavior of dropdown menus on native platforms.

I'm not sure we can handle this case internally without creating a delay on the open state to try and determine the user's intent to drag, which isn't a great UX for the vast majority of users. This would also pose some accessibility challenges.

You should be able to work around this by controlling the state yourself and only calling the change handler when the user has completed a click. Might add a bit of complexity to your code, but I'm not convinced this should be baked into the component's behavior.

**Comment by BrooceLR** [comment]
Please note this causes every touchscreen drag interaction starting over a dropdown to accidentally open it.
You can see the issue by simply visiting the [dropdown docs](https://www.radix-ui.com/primitives/docs/components/dropdown-menu) on a mobile device and start dragging from the dropdown handle.

Maybe there's an accessibility reason for using `onPointerDown` instead of `onClick` to toggle the open state of it?

If not, hope you'll consider reopening the issue
